src/syrupy/report.py

```diff
@@ -67,6 +75,43 @@ def __attrs_post_init__(self) -> None:
                 else:
                     self.failed.update(snapshot_fossil)
 
+    def __parse_invocation_args(self) -> None:
```

Comment:
Complexity has grown quite a bit. I'm wondering if we should do a pass afterwards and ensure sufficient docstrings to make contributing to syrupy easier

Reply:
Makes sense

Reply:
Added